drivers/i2s/i2s_ll_stm32.h

```diff
@@ -58,4 +61,16 @@ struct i2s_stm32_data {
 	struct stream tx;
 };
 
+/* checks that DMA Tx packet is fully transmitted over the I2S */
+static inline uint32_t ll_func_i2s_dma_busy(SPI_TypeDef *i2s)
```

Comment:
```suggestion
static inline uint32_t ll_func_i2s_dma_busy(I2S_TypeDef *i2s)
```
?

Reply:
The typdef of I2S does not exist in the hal_stm32, we used the spi type def in the structure i2s
```c
/* Device constant configuration parameters */
struct i2s_stm32_cfg {
	SPI_TypeDef *i2s;
	const struct stm32_pclken *pclken;
	size_t pclk_len;
	const struct pinctrl_dev_config *pcfg;
```

Reply:
Why is this function not in the source file?